randovania/games/prime1/logic_database/header.txt

```diff
@@ -256,5 +256,11 @@ Dock Weaknesses
           Trivial
       No lock
 
+
+  * Frigate Ship Teleporter
+      Open:
+          Scan Visor and After Parasite Queen
```

Comment:
how so?

Reply:
Ah, you don't need to scan the ship during the escape?

Reply:
The only Scan requirements after fighting Parasite Queen are two elevators, one right after the fight and one later on where you normally lose your items. Entrance randomizer could circumvent these requirements.

Reply:
The Scan visor requirement here isnt an indirect requirement due to escaping queen.  
The reason the scan requirement is here is, because to my knowledge you need to scan *the ship* in order to get teleported.
If my knowledge there is false, I'll remove the scan requirement.

Reply:
The transport happens when you enter through the last door in Exterior Docking Hanger after the escape sequence is activated. Unless something was changed recently, you do not scan the ship to get transported.

Reply:
Based on the answer [here](https://discord.com/channels/914291389293027329/1268394331488911440/1268443408326725682) I understood it that you can also get teleported without explicitly triggering the escape sequence, as long as you kileld PQ once before.

Reply:
I wasn't aware of that, but looks like both are options in this case. Writing the logic around it is pretty tricky, so it should be fine to have the scan visor requirement for now since you'll need it anyways and we can take another look at the logic in the future once/if entrance rando gets logic.

Reply:
Then i'll remove the scan req